CMake/resolve_dependency_modules/README.md

```diff
@@ -32,11 +32,12 @@ by Velox. See details on bundling below.
 | re2               | 2021-04-01      | Yes      |
 | fmt               | 10.1.1          | Yes      |
```

Comment:
I thought this folly version also uses fmt11? Or is it backwards compatible?

Reply:
It is backwards compatible. I had originally planned to also upgrade fmt to 11 but there is another PR that needs to be merged first that prepares the code for fmt 11.
Upgrading fmt is next.

Reply:
It is backward compatible since Meta is still on fmt9.